src/components/KToaster/KToaster.vue

```diff
@@ -41,76 +61,128 @@ defineProps({
 defineEmits<{
   (e: 'close', val: any): void
 }>()
+
+const getToastIcon = (appearance?: ToasterAppearance): ToastIcon => {
+  switch (appearance) {
+    case 'info':
+      return InfoIcon
+    case 'success':
+      return CheckCircleIcon
+    case 'warning':
+      return WarningIcon
+    case 'danger':
+      return ClearIcon
+    case 'system':
+      return KongIcon
+    default:
+      return InfoIcon // info as default in case of invalid appearance
+  }
+}
 </script>
 
 <style lang="scss" scoped>
-@import '@/styles/tmp-variables';
-@import '@/styles/styles';
-
-$transition: all .3s;
-
-.toaster-container-outer {
+.k-toaster {
   bottom: 16px;
-  max-width: 300px;
+  display: flex;
+  flex-direction: column;
+  gap: var(--kui-space-50, $kui-space-50);
   position: fixed;
   right: 16px;
-  transition: $transition;
-  width: auto;
+  width: 400px;
   z-index: 10000;
```

Comment:
suggestion: can we expose the `z-index` as a prop? Oftentimes you may need to override depending on the structure of the rest of your app. You can use this value as the default

Reply:
Same as in the other thread - we're not really exposing `z-index` as prop anywhere else so it's weird? Maybe leave that to CSS?

Reply:
But why not? We expose max width, isn't it the same concept?

Reply:
My thinking: let's say you have the intercom widget that is zindex of 20000 - you'd have to globally override somewhere in your app instead of just passing it as a prop to override

Reply:
How about this: I'll handle adding zIndex in all components that need it (KModal, KTooltip, KToaster, KSlideout) in a separate PR?

Reply:
works for me; just throw a ticket in the backlog; we can address it later if/when it becomes an issue (doesn't need to be part of the reskin effort)